egs/aishell/s10/RESULTS

```diff
@@ -0,0 +1,45 @@
+# If you execute `run.sh`, then you should get similar results as follows:
+
+# Results for kaldi pybind LF-MMI training with PyTorch
+## head exp/chain/decode_res/*/scoring_kaldi/best_* > RESULTS
+#
+==> exp/chain/decode_res/dev/scoring_kaldi/best_cer <==
```

Comment:
The naming scheme is not obvious from this file... what is "res"?  Please clarify this, and also chain_nnet3.
And can you please make sure that these results (and where appropriate, the output of chain_dir_info.pl) are 
in a comment at the top of the script that generated them?

Reply:
thanks, I will change it to follow the current style of `egs/swbd/s5c/RESULTS`.